.github/workflows/build-publish-containers.yml

```diff
@@ -58,71 +57,23 @@ jobs:
       - name: Convert and store repository owner in lowercase
         run: |
           echo REPOSITORY_OWNER=$(echo ${{ github.repository_owner }} | tr '[:upper:]' '[:lower:]') >> $GITHUB_ENV
-          echo DOCKERFILE=${{ matrix.tag }}/Dockerfile >> $GITHUB_ENV
+          echo IMAGE_TAG=$(date +"%F")-${{ github.GITHUB_RUN_ID }} >> $GITHUB_ENV
 
       - name: Build and push to GitHub Packages
         uses: docker/build-push-action@v3
         with:
-          tags: ghcr.io/${{ env.REPOSITORY_OWNER }}/${{ matrix.tag }}-${{ matrix.platform }}
-          file: ${{ env.DOCKERFILE }}
-          platforms: linux/${{ matrix.platform }}
+          tags: |
```

Comment:
Getting the grip of github actions now. :D 
Using two tags seems a good idea here.
1. `:latest` is going to be overwritten every time a new build is done. 
2.  The image will also be published as `$(date +"%F")-${{ github.GITHUB_RUN_ID }}` to keep it around to reproduce errors and have a lasting reference.